_data/students.yml

```diff
@@ -282,4 +282,11 @@
   image: JLeow.png
   facebook: jerome.leow.5
   twitter: JLeow00
-  bio: Ukulele is love, ukulele is life
\ No newline at end of file
+  bio: Ukulele is love, ukulele is life
+  
+- name: Ching Hannie
+  github: ChingHannie
+  image: ching.jpg
+  facebook: https://www.facebook.com/abchingh
```

Comment:
We don't need the whole url. Just your username would be enough. Look at other entries in the file.